test/spell_check.words

```diff
@@ -1,25 +1,24 @@
-app
+ament
+apache
 argcomplete
-args
-basepath
-bool
-buildfile
 colcon
-config
-cpu
-dict
+completers
 esteve
-fhs
```

Comment:
Was this removed because it is no longer in the codebase or because it's in the upstream dictionary for scspell?

Reply:
I don't see `fhs` appearing anywhere in this repository. I believe it was probably an artifact from copy/paste of another colcon extension.